proto/band/tunnel/v1beta1/tunnel.proto

```diff
@@ -0,0 +1,139 @@
+syntax = "proto3";
+package band.tunnel.v1beta1;
+
+import "amino/amino.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+
+import "band/feeds/v1beta1/feeds.proto";
+
+option go_package            = "github.com/bandprotocol/chain/v3/x/tunnel/types";
+option (gogoproto.equal_all) = true;
+
+// TSSRoute is the type for a TSS route
+message TSSRoute {
+  option (cosmos_proto.implements_interface) = "RouteI";
+
+  // destination_chain_id is the destination chain ID
+  string destination_chain_id = 1 [(gogoproto.customname) = "DestinationChainID"];
+  // destination_contract_address is the destination contract address
+  string destination_contract_address = 2;
+}
```

Comment:
I think we should separate XXXRoute and XXXPacketContent into separate files like route.proto. to separate between core logic and route type. wdyt? 

Anyway, I'm ok with both. can just move them to be near each other.